tests/foreman/sys/test_custom_cert_installation.py

```diff
@@ -0,0 +1,120 @@
+"""Test class for ``custom-cert-installation``.
+
+:Requirement: custom-cert-installation
+
+:CaseAutomation: Automated
+
+:CaseLevel: System
+
+:CaseComponent: Certificates
+
+:Assignee: okhatavk
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+
+"""
+import pytest
+
+from robottelo.config import settings
+from robottelo.ssh import get_connection
+
+
+@pytest.mark.tier1
+def test_custom_cert_update_satellite(satellite_latest, custom_certs_factory):
+    """Update the satellite instance with new custom certs.
+
+    :id: 6408226e-6c7f-4394-8a60-fb86f058f3fd
+
+    :steps:
+        1. Generate the custom certs
+        2. Update satellite with custom certs
+        3. Assert output does not report SSL certificate error
+        4. Assert all services are running
+
+    :expectedresults: Satellite should be updated with custom certs
+    """
+    satellite = satellite_latest.hostname
+    cert_data = custom_certs_factory(hostname=satellite)
+    with get_connection(hostname=satellite) as connection:
+        result = connection.run(
+            'katello-certs-check -c {} -k {} -b {}'.format(
+                cert_data['cert_file_name'],
+                cert_data['key_file_name'],
+                cert_data['ca_bundle_file_name'],
+            ),
+            output_format='plain',
+        )
+        assert result.return_code == 0, 'Failed to generate the custom cert check command'
+
+        # update the custom certs in Satellite
+        command = f'satellite-installer --scenario satellite \
```

Comment:
Please use the new InstallerCommand class to build this installer command:

https://github.com/SatelliteQE/robottelo/blob/4430782b17b33fceda28e52af84b69fdecbdaa85/robottelo/helpers.py#L718